staging/src/k8s.io/cli-runtime/pkg/resource/selector.go

```diff
@@ -77,6 +80,11 @@ func (r *Selector) Visit(fn VisitorFunc) error {
 		if err := fn(info, nil); err != nil {
 			return nil, err
 		}
+
+		if fmt.Sprintf("%v",reflect.Indirect(reflect.ValueOf(list)).FieldByName("Items"))=="[]" {
+			fmt.Fprintf(os.Stderr, "No resources found in %s namespace\n",r.Namespace)
+		}
```

Comment:
@ardaguclu @eddiezane  I am not sure how optimal of a solution this is. This is what i currently came up with without changing the implementation in rollout-status.go. I struggled with handling the different types of lists that can be found here. Ex: v1.DeploymentList, v1.ReplicaSetList. Thanks for your input.

Reply:
Upon more investigation this is not a complete implementation as it will not catch all cases and also can interfere with the outputs of other commands. Following the get command we could use the trackingWriter to write no resources if nothing has been written out during the rollout-status command. Is that the more standard approach? Thanks